tools/dotnet-fssrgen/project.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
remove that file, see https://github.com/fsharp/FSharp.Compiler.Service/pull/556#issuecomment-212895651